prerequisites.md

```diff
@@ -1,88 +1,51 @@
-  - [Git for windows](#getgit) allows you to run git commands, through a bash shell. There is even a portable version that does not require admin rights.
-  - Many more options. You can use whatever you're comfortable with. _(note that the format in which PuttyGen stores the keys is _not_ standard; you will need to copy the generated key in a new text file)_
-
-WSL has our preference, with Git for Windows acting as a backup-option. Guide for both options have been included.
+- [UMP](#glossary), to be able to interact with the [DSH](#glossary)
 
-<a name="powershellkeygen"></a>
+Before the training starts, you will receive:
 
-## Generating a keypair in Windows PowerShell
+- The IP-address of your [virtual machine (VM)](#glossary), which will be created for the purposes of this training
+- A private SSH key. This key is part of a [keypair](#glossary), with the other part being given to your VM
 
-1. In PowerShell, type the following commands (press ENTER after each command)
+If you are already familiar with SSH, have a working shell or client, then you can skip straight to your preferred version of 'connecting'.
 
-```powershell
-mkdir %userprofile%/.ssh
-cd %userprofile%/.ssh
-ssh-keygen -t rsa -C "your_email@example.com"
-```
-
-2. Follow the instructions. Note that you will have to set a password.
-3. It will save the key in the folder %userprofile%/.ssh. Use a file explorer to navigate to this folder.
-4. Mail the [public key](##glossary) file to the trainer.
+## Options to connect over SSH
 
-The rest of the instructions are the same as those under [Using WSL](#usewsl)
+- Linux and Mac have an SSH client built in. [Proceed to Generating a keypair in Linux/Mac](#using-linux-or-mac).
+- Windows sometimes comes with ssh. To check, you open Powershell (WIN+X, A, click `yes`) and type `ssh`. You will either get feedback stating:
+  - you didn't supply required parameters. This means Powershell understands SSH, and you can proceed to treat it as a relatively normal [Linux shell](#using-wsl) for the commands that we use here. However, if commands result in unexpected behaviour, you should switch to one of the alternatives below.
+  - ssh is unknown, meaning PowerShell does not understand SSH. you should then proceed to install one of the following:
+    - [Windows Subsystem for Linux (WSL)](#getting-wsl) allows you to run Linux commands (including SSH) on your Windows system, by running an integrated [Linux shell](#using-wsl).
+    - [Git for windows](#getting-git-bash) allows you to run git commands, through a [bash shell](#using-git-bash). There is even a portable version that does not require admin rights.
+    - If already have another preferred way, such as Putty, you are free to use it, assuming it does not create issues during the training. _(note, however, that the format in which PuttyGen stores the keys is __not__ standard; you will need to copy the generated key in a new text file)_
```

Comment:
Store generated key? Didn't you edit this guide because we intend to provide keys to participants?

Reply:
Agreed, removed.